pegasus/sites.v3/code.org/public/curriculum/csf.haml

```diff
@@ -37,7 +37,7 @@ theme: responsive_full_width
     .clear
 
 %section.bg-neutral-light
-  - languages = "العربية, Bahasa Indonesia, Català, 简体字, 繁體字, Čeština, Français, Deutsch, हिन्दी, Italiano, 日本語, 한국어, ಕನ್ನಡ, بهاس ملايو, Монгол хэл, Polski, Português (Brasil), Română, Русский, Slovenčina, Filipino, தமிழ், ภาษาไทย, Türkçe, Українська, Español (España), Español (LATAM), اردو, O'zbekcha, Tiếng Việt"
+  - languages = "فارسی ,العربية , Bahasa Indonesia, Català, 简体字, 繁體字, Čeština, Français, Deutsch, हिन्दी, Italiano, 日本語, 한국어, ಕನ್ನಡ, بهاس ملايو, Монгол хэл, English, Polski, Português (Brasil), Română, Русский, Slovenčina, Filipino, தமிழ், ภาษาไทย, Türkçe, Українська, Español (España), Español (LATAM), اردو, O'zbekcha, Tiếng Việt"
```

Comment:
I checked with @castro-jorge in the past and he said English didn't need to be added here.

Reply:
I think we should list all the languages including English

Reply:
I agree re: including English.